org.jacoco.tests/pom.xml

```diff
@@ -33,6 +33,7 @@
     <module>../org.jacoco.ant.test</module>
     <module>../jacoco-maven-plugin.test</module>
     <module>../org.jacoco.examples.test</module>
+    <module>../org.jacoco.tests.coverage</module>
```

Comment:
@Godin I was thinking of a new module org.jacoco.build.reports where we create all the aggregate reports (JavaDoc, JUnit, Coverage). Let's handle this in a separate PR
Indeed you're right, we might need to consolidate multiple dependencies to the same project with the `report-aggregate` goal. Again I would handle this in a separate PR when we learn how the new goal works in real life projects.